R/unnest.R

```diff
@@ -0,0 +1,186 @@
+#' Unnest a 2d list-column into rows and columns
+#'
+#' Unnest expands a list-column containing 2d data structures (e.g. data frames)
+#' into rows and columns.
+#'
+#' @inheritParams unchop
+#' @inheritParams unpack
```

Comment:
If you look at the `?unnest` page, the `...` docs are actually coming from `pack()`

I think you want to mention that `...` are deprecated, maybe you can pull from the current dots of `nest()`